dimod/template_sampler.py

```diff
@@ -104,7 +104,7 @@ def sample_qubo(self, Q, **kwargs):
                 sample_ising.
 
         Returns:
-            :obj:`BinaryResponse`:
+            :obj:`.BinaryResponse`:
```

Comment:
Why does this require a leading dot?

Reply:
Because in order for sphinx to make the link it needs to know to look in `dimod.BinaryResponse`

Reply:
I did this change with Alex - the hotlink wasn't working otherwise

Reply:
Oh so `.BinaryResponse` is short hand for `dimod.BinaryResponse`? Neat-o

Reply:
I told it where to look in the .rst file `.. currentmodule:: dimod`. Or at least I am pretty sure that's how that works...